File-Formats/DocIO/Working-with-Tables.md

```diff
@@ -966,6 +966,139 @@ private void AlignCellContentForTable(WTable table, Syncfusion.DocIO.DLS.Horizon
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Tables/Align-text-within-a-table).
 
+## Insert Image to Table Cell
+
+You can insert an image into table cell by accessing the paragraph in that cell. The following code example illustrates how to insert an image into table cell.
+
+{% tabs %} 
+
+{% highlight c# tabtitle="C#" %}
+//Creates an instance of WordDocument class.
+using (WordDocument document = new WordDocument())
+{
+    IWSection section = document.AddSection();
+    //Adds a new table into Word document.
+    IWTable table = section.AddTable();
+    //Specifies the total number of rows & columns.
+    table.ResetCells(2, 2);
+    table[0, 0].AddParagraph().AppendText("Product Name");
+    table[0, 1].AddParagraph().AppendText("Product Image");
+    table[1, 0].AddParagraph().AppendText("Apple Juice");
+    //Adds the image into cell.
+    IWPicture picture = table[1, 1].AddParagraph().AppendPicture(Image.FromFile("Image.png"));
+    picture.Height = 75;
+    picture.Width = 60;
+    //Saves the Word document.
+    document.Save("Sample.docx", FormatType.Docx);
+}
+{% endhighlight %}
+
+{% highlight vb.net tabtitle="VB.NET" %}
+'Creates an instance of WordDocument class.
+Using document As New WordDocument()
```

Comment:
Using the document As New WordDocument()

Reply:
It's VB code. So it is not possible to change